bundles/AdminBundle/Controller/Admin/TranslationController.php

```diff
@@ -51,25 +51,38 @@ class TranslationController extends AdminController
     public function importAction(Request $request)
     {
         $admin = $request->get('admin');
+        $dialect = $request->get('csvSettings', null);
+        $tmpFile = $request->get('importFile');
```

Comment:
And here we need to validate the filename and append it to the known path (`PIMCORE_SYSTEM_TEMP_DIRECTORY`).